sdk/include/opentelemetry/sdk/metrics/meter.h

```diff
@@ -105,7 +106,11 @@ class Meter final : public opentelemetry::metrics::Meter
   // order of declaration is important here - instrumentation library should destroy after
   // meter-context.
   std::unique_ptr<sdk::instrumentationlibrary::InstrumentationLibrary> instrumentation_library_;
-  std::shared_ptr<sdk::metrics::MeterContext> context_;
+  std::shared_ptr<sdk::metrics::MeterContext> meter_context_;
+  std::unordered_map<std::string, std::shared_ptr<MetricStorage>> storage_registry_;
```

Comment:
It would be nice to have the key documented here.

Reply:
Yes, have added the documentation now.